Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -66,20 +66,25 @@ open class ChatThreadVC:
         }
 
         messageController.delegate = self
-        let updateMessageControllerMessage: () -> Void = { [messageController, messageComposerVC] in
+
+        let completeSetup: (ChatMessage?) -> Void = { [messageController, messageComposerVC] message in
```

Comment:
```suggestion
        let completeSetUp: (ChatMessage?) -> Void = { [messageController, messageComposerVC] message in
```

Better to have the same camel case of `setUp` so that is easier to understand what the variable means.

Reply:
done